CMakeLists.txt

```diff
@@ -82,3 +82,4 @@ target_link_libraries(cosim
 
 install(TARGETS cosim RUNTIME DESTINATION bin)
 include(InstallRequiredSystemLibraries)
+install(FILES "LICENSE" "README.md" DESTINATION "doc")
```

Comment:
I can't see that this step works. It's been a long time since I played with the CLI, is there a special install command I should run?

Reply:
Yes, build the `install` target (either in your IDE or from the command line with `cmake --build <build dir> --target install` – or `make install` on Linux). Then the files will be installed to `<build dir>/dist/doc`, alongside the library and executable directories. It's how we put together our release bundle.

Reply:
Arigato, sensei!